templates/mountingEdit.html

```diff
@@ -11,12 +11,12 @@
                 <div class="row">
                     <!--mounting information-->
                     <form id="edit-mounting-form" name="edit-mounting-form" action="/editMounting" method="post">
-                        <h4><b>Module Code:</b></h4>
-                        <p>$moduleCode</p>
+                        <label for="module-code">Module Code:</label>
+                        <p id="module-code">$moduleCode</p>
```

Comment:
Inconsistency still present:

`<h1 class="row title text-center">Edit: $moduleCode</h1>` was used as the edit-module header.

Please follow this format for edit-mounting header, except that the header value can be `Edit Mounting for $moduleCode`. Perhaps `Edit General Info for $moduleCode` can be used for the edit-module header too.